contents/blog/posthog-vs-mixpanel.md

```diff
@@ -0,0 +1,968 @@
+        </tr>
+        <tr>
+            <td><strong>React Native</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>PostHog: Session recording not available</td>
+        </tr>
+        <tr>
+            <td><strong>Flutter</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>PostHog: Event tracking and user identification only</td>
+        </tr>
+        <tr>
+            <td><strong>iOS (Swift)</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>PostHog: Autocapture and session recording not available</td>
+        </tr>
+        <tr>
+            <td><strong>iOS (Objective-C)</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>PostHog: Autocapture and session recording not available</td>
+        </tr>
+        <tr>
+            <td><strong>Android</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>PostHog: Event tracking and user identification only</td>
+        </tr>
+        <tr>
+            <td><strong>Unity</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Event tracking for game developers</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+- **JavaScript snippet:** All client libraries support event tracking and user identification for product analytics, but we recommend using PostHog's JavaScript snippet to enjoy all our features. Our iOS libraries also support feature flagging and group analytics. See our [client library documentation](/docs/integrate/client/snippet-installation) for more information.
+
+- **Mobile app session recordings:** We don't currently support session recording in mobile apps, but it's currently under consideration as a project for our session recording team. See our [public roadmap](/roadmap) for more info.
+
+- **Unity:** We don't have a Unity SDK yet but, as we're open-source, anyone can create one. See our [contribution guidelines](/docs/contribute) for more info.
+
+### Server libraries
+
+Some PostHog features, such as session recording and autocapture, are only available via our client-side libraries.
```

Comment:
This should be a note below the table to be consistent with client libraries IMO